specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1052,6 +1099,11 @@ components:
           description: "Dit element is de identificatie van het Stuk. Dit kan een aangeboden Stuk of een Kadasterstuk zijn."
           items:
             type: string
+        zakelijkGerechtigdeIdentificatie:
```

Comment:
meervoud (is een array):  zakelijkGerechtigdeIdentificaties of  zakelijkGerechtigdenIdentificaties of  zakelijkGerechtigdenIdentificatie?

Reply:
is zakelijkGerechtigdeIdentificaties geworden